baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -351,7 +328,11 @@ export class AssetServiceClient {
  *   Call options. See {@link https://googleapis.dev/nodejs/google-gax/latest/interfaces/CallOptions.html|CallOptions} for more details.
  * @returns {Promise} - The promise which resolves to an array.
  *   The first element of the array is an object representing [BatchGetAssetsHistoryResponse]{@link google.cloud.asset.v1.BatchGetAssetsHistoryResponse}.
- *   The promise has a method named "cancel" which cancels the ongoing API call.
+ *   Please see the
+ *   [documentation](https://github.com/googleapis/gax-nodejs/blob/master/client-libraries.md#regular-methods)
```

Comment:
@fhinkel @tbpg I think we need to spend some time thinking through what we want generators to do here :)

Reply:
Here I really want to stop advertising the useless `.cancel()` method in hope that we may be able to drop it later, like in 2 years or so :)